docs/plugins/for_plugin_developers.rst

```diff
@@ -327,6 +327,19 @@ If you are using Github, add the `"napari-plugin" topic
 <https://github.com/topics/napari-plugin>`_ to your repo so other developers can
 see your work.
 
+The `napari hub <https://www.napari-hub.org/>`_
+automatically displays information about all napari plugins,
```

Comment:
"napari plugins served on pypi?"  (since valid napari plugins can also come from conda?  but I don't think they're on the hub yet right?

Reply:
Yeah in this case it's even more important, since developers are the ones who will actually need this information.